components/Projects/ProjectCard/AvatarList.tsx

```diff
@@ -0,0 +1,23 @@
+import Avatar, { AvatarData } from './Avatar';
+
+export interface AvatarListProps {
+  avatars: AvatarData[];
+  className?: string;
+}
+
+export default function AvatarList({
+  avatars: avatars,
+  className = '',
+}: AvatarListProps) {
+  return (
+    <div className={`flex gap-2 ${className}`}>
```

Comment:
try using the `classNames` like i did in `Carousel.tsx`. I already spoke with @RoyTennenbaum about making it our convention for dynamic classes because there we can define classes with static strings, dynamic strings, objects.
in this case it would look like -> `<div className={classNames('flex gap-2', className)}>`